src/commands/DocContent.ts

```diff
@@ -0,0 +1,115 @@
+import * as path from "path";
+import * as fs from "fs"
+import * as vscode from 'vscode'
+import * as parser from "../parser"
+
+export const id = "show_doc";
+var show_doc_item_selector: boolean = true;
+
+export default class DocContent {
```

Comment:
Rename the name of this class to BuiltinDocHelper or something to describe the usage of the class.